From ea690e3d324b79f0b6e91411da60bb8b0f5dee4e Mon Sep 17 00:00:00 2001
From: Brahma Reddy Battula <brahma@apache.org>
Date: Tue, 27 Sep 2016 10:45:12 +0530
Subject: [PATCH 2081/2748] HDFS-9885. Correct the distcp counters name while
 displaying counters. Contributed by Surendra
 Singh Lilhore

(cherry picked from commit e17a4970bea8213660bb6c550104783069153236)
(cherry picked from commit ff4fb48da1068e0b19a2b1e55ce20b6c800f7893)

Change-Id: I9f7b86f1a2654899e0b6f78ee64eeb94a40f4487
---
 .../tools/mapred/CopyMapper_Counter.properties     |   24 ++++++++++++++++++++
 1 file changed, 24 insertions(+)
 create mode 100644 hadoop-tools/hadoop-distcp/src/main/resources/org/apache/hadoop/tools/mapred/CopyMapper_Counter.properties

diff --git a/hadoop-tools/hadoop-distcp/src/main/resources/org/apache/hadoop/tools/mapred/CopyMapper_Counter.properties b/hadoop-tools/hadoop-distcp/src/main/resources/org/apache/hadoop/tools/mapred/CopyMapper_Counter.properties
new file mode 100644
index 0000000..53727ee
--- /dev/null
+++ b/hadoop-tools/hadoop-distcp/src/main/resources/org/apache/hadoop/tools/mapred/CopyMapper_Counter.properties
@@ -0,0 +1,24 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# ResourceBundle properties file for DistCp counters
+
+CounterGroupName=         DistCp Counters
+
+COPY.name=                Files Copied
+SKIP.name=                Files Skipped
+FAIL.name=                Files Failed
+BYTESCOPIED.name=         Bytes Copied
+BYTESEXPECTED.name=       Bytes Expected
+BYTESFAILED.name=         Bytes Failed
+BYTESSKIPPED.name=        Bytes Skipped
+BANDWIDTH_IN_BYTES.name=  Bandwidth in Btyes
\ No newline at end of file
-- 
1.7.9.5

